### PR TITLE
Onboarding tour added

### PR DESCRIPTION
**Pull request by Tehsurfer**
Live demo at:
https://map-tutorial.herokuapp.com/

### CODE REVIEW

File: src/components/FloatingFlow.vue [hunk 1]
```diff
@@ -170,14 +182,78 @@ export default {
...
+      var hasVisted = localStorage.getItem('hasVisitedMaps')
+      if (hasVisted === null || hasVisted === false){
+          localStorage.setItem('hasVisitedMaps', true);
+          this.startTutorial()
+      }
+    },
+    startTutorial: function(){
+      this.tour.start()
+    },
+    previousStepCallback: function(currentStep){
```
[the remark this answers is not in the document]
Comment (Tehsurfer) [reply src/components/FloatingFlow.vue:207]: Mmmm not really, happy to remove it

File: src/components/MapPopover.vue [hunk 2]
```diff
@@ -62,6 +62,10 @@ export default {
       }
     return false;
     },
+    setTooltipCoords: function(x, y){
```
Comment (alan-wu) [inline src/components/MapPopover.vue:65]: You should not change values of props in a component, either change it on the parent component or change the data "tStyle". Take at look at the watch routine.

**Comment by alan-wu** [comment]
There still missing semicolons at some places including the new file.

File: src/components/MapPopover.vue (hunk 2 above)
Comment (Tehsurfer) [reply src/components/MapPopover.vue:65]: Yep algood I will switch this to FloatingDialog now.

**Pull request merged**